src/components/Party/Party.vue

```diff
@@ -63,18 +65,29 @@ export default {
           break;
       }
     },
-    movePlayer(player, position) {
-      const players = this.players.map(pl => pl)
-      const playerIndex = players.indexOf(player)
-      players[playerIndex] = { ...player, x: position.x, y: position.y }
-
-      this.$set(this.$data, 'players', players)
+    async movePlayer(player, position) {
+      try {
+        const URL = 'https://wars.quiches.ovh/api/party/action'
```

Comment:
you can use variable from .env file to set URL

Reply:
API_BASE_URL is throwing undefined